Makefile

```diff
@@ -117,8 +117,8 @@ start-docker: ## Starts the docker containers for local development.
 ifeq ($(IS_CI),false)
 	@echo Starting docker containers
 
-	docker-compose --no-ansi run --rm start_dependencies
```

Comment:
Wouldn't it make more sense to prevent ansi colors on the CI branch?

Reply:
Note that we don't actually use `start-docker` for CI -- e.g. this skips this block: `ifeq ($(IS_CI),false)`. So there's no need for `--no-ansi` in this case.